nap/utils.py

```diff
@@ -60,19 +54,21 @@ def flatten_params(k, vs):
 
         # since we can have more than one value for a single key, we use a
         # tuple of two tuples instead of a dictionary
-        params_tuple = tuple(itertools.chain(*flat_params))
-        param_string = urllib.parse.urlencode(params_tuple)
+        params_tuple = tuple(sorted(itertools.chain(*flat_params)))
```

Comment:
This seems to be skipping some normalization that the other logic was handling, see https://docs.python.org/2/library/urlparse.html#urlparse.urlunsplit. Is that going to allow more than one equivalent URL to exist that will create multiple cache entries for the same resource?